level2/asg.tf

```diff
@@ -0,0 +1,8 @@
+module "vpc" {
```

Comment:
could you please change the module name as it is bearing vpc which have already been defined.